src/components/DashboardWidgetItem/index.js

```diff
@@ -0,0 +1,3 @@
+import DashboardWidgetItem from './DashboardWidgetItem.vue'
+
+export default DashboardWidgetItem
```

Comment:
Should it even be exported?
Anyone using it directly instead of via the default `DashboardWidget` ?

Reply:
Yes, it's needed. Otherwise you don't have access to the `avatar` slot of `DashboardWidgetItem`, which e.g. Calendar and Tasks use to show the calendar indicator dot instead of the avatar.
Also, removing it would be a breaking change, and I think a drop-in replacement is nicer. Then you just need to adjust the imports and be done.